index.html

```diff
@@ -94,7 +95,7 @@ <h3>Multi-Post Stories Gain+Glory</h3>
 						</ul>
 					</div>
 					<div class="card-see-prjct">
-						<button class="green-button" type="button">See Project</button>
+						<button class="green-button cProject" id="cP1" type="button">See Project</button>
 					</div>
 					</div>
 			</div>

```

Comment:
- [ ] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.**

Reply:
Thanks for the suggestion, I made the works-section content generate dynamically on javascript.
Added a "reducedBadges" list on the `popupData` that contains the main badges of each card so as to keep the card style consistent